papers/ring_vrf/overview.tex

```diff
@@ -175,28 +176,28 @@ \section{Protocol overview}
 	}
 } $$
 
-We discovered this becomes incredibly efficient if one specializes
-the Groth16 construction:  An inner true Groth16 for $\Lring^\inner$
+We discovered the SNARK for the language \Lring becomes incredibly efficient for the prover if one specializes
+the original Groth16 SNARK construction:  An inner original Groth16 SNARK for $\Lring^\inner$
 handles the secret key \sk directly via its public inputs, but
 \sk and even \pk remain secret by transforming the trusted setup to have
-a rerandomizable Pedersen commitment \compk outside this Groth16.
+a rerandomizable Pedersen commitment \compk outside this SNARK.
```

Comment:
It does kinda matter that it's groth16 here imho

Reply:
OK, I have updated to "this Groth16 SNARK".